src/go/k8s/kuttl-v2-test.yaml

```diff
@@ -3,6 +3,8 @@ kind: TestSuite
 startKIND: true
 kindContainers:
   - localhost/redpanda-operator:dev
+#  - docker.redpanda.com/redpandadata/redpanda:v23.2.8
+#  - docker.redpanda.com/redpandadata/console:v2.3.2
```

Comment:
Why those 2 comments are added?

Reply:
for local testing, reminder you can preload these instead of downloading them each time.